NBitcoin/UInt2561.cs

```diff
@@ -1,11 +1,13 @@
 ﻿

```

Comment:
For uint256 and uint160, this branch and master have both implemented IEquatable and IComparable,
both seems to work fine. But lets just have minimum change here. So would you remove this file from this PR? @knocte 

It should be no functional difference.

Reply:
done